index.js

```diff
@@ -2485,17 +2598,14 @@
                def);
   }
 
-  var create =
-  def('create',
-      {},
-      [RecordType({checkTypes: Boolean_, env: Array_(Any)}), AnyFunction],
-      _create);
+  var Options = RecordType(Object_, {checkTypes: Boolean_, env: Array_(Type)});
+  var create = def('create', {}, [Options, AnyFunction], _create);
```

Comment:
I'd prefer this arrangement to make it clear that the record type is not used elsewhere:

```javascript
  var create =
  def('create',
      {},
      [RecordType(Object_, {checkTypes: Boolean_, env: Array_(Type)}),
       AnyFunction],
      _create);
```